### ISSUE TITLE
Getting issue while using Modal

### ISSUE THREAD

**Issue by ikismail** [akveo/react-native-ui-kitten#873]
## 🐛 Bug Report
Getting rendering issue while using Modal component with Backdrop, Followed the Documentation [Modal Documentation](https://akveo.github.io/react-native-ui-kitten/docs/components/modal/overview#modal)

![image](https://user-images.githubusercontent.com/17190913/74104914-768b6c80-4b7f-11ea-8130-f67c14cab917.png)

## Expected behavior

Toggling modal without any rendering issues

## Link to runnable example or repository (highly encouraged)

https://gist.github.com/ikismail/c14318c1d80fbe4e14fd32d54c792927

## UI Kitten and Eva version

| Package      | Version |
| ----------- | ----------- |
| @eva-design/eva      |    ^1.3.2    |
| @ui-kitten/components   |    4.4.0-beta.3     |

## Environment information

```
System:
    OS: macOS 10.15.3
    CPU: (4) x64 Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz
  Binaries:
    Node: 10.16.3 - /usr/local/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 6.13.4 - /usr/local/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5977832
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild

```

**Comment by artyorsh** [comment]
Reported in #847 and fixed in #854 and will be available in next release

**Comment by Ramana-G** [comment]
Still, I am able to reproduce the issue. is the issue resolved?
